package-lock.json

```diff
@@ -1,15 +1,15 @@
 {
     "name": "ontology-starter-react-app",
     "version": "0.0.0",
-    "lockfileVersion": 2,
+    "lockfileVersion": 3,
```

Comment:
Did you intentionally upgrade the lockfile version?
https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json#lockfileversion

Reply:
Yes, deleted the old on and re-created this so we don't have any reference to swirl in the file